optuna/storages/grpc/api.proto

```diff
@@ -0,0 +1,436 @@
+/**
+ * Optuna GRPC API
+ * The following command generates the Python code from this file:
+ $ python -m grpc_tools.protoc \
+    --proto_path=optuna/storages/grpc \
+    --grpc_python_out=optuna/storages/grpc/_auto_generated \
+    --python_out=optuna/storages/grpc/_auto_generated \
+    --mypy_out=optuna/storages/grpc/_auto_generated \
+    optuna/storages/grpc/api.proto
```

Comment:
How about leaving a sed command to replace an import statement?

```suggestion
 $ python -m grpc_tools.protoc \
    --proto_path=optuna/storages/grpc \
    --grpc_python_out=optuna/storages/grpc/_auto_generated \
    --python_out=optuna/storages/grpc/_auto_generated \
    --mypy_out=optuna/storages/grpc/_auto_generated \
    optuna/storages/grpc/api.proto
 $ sed -i \
    's|import api_pb2 as api__pb2|import optuna.storages.grpc._auto_generated.api_pb2 as api__pb2|' \
    optuna/storages/grpc/_auto_generated/api_pb2_grpc.py
```

Reply:
It is also helpful to remember the following toolkit versions used to generate the code for reproducibility.
- protobuf
- grpcio-tools
- mypy-protobuf